### PR TITLE
Batch export ROIs from screen

### PR DESCRIPTION
**Pull request by will-moore**
I found this useful recently, exporting big CSV from many images in a Screen or Plate.

To test:
 - Create lots of ROIs on many images in a Screen or Project.
 - Try to Export ROIs via this script from Screen, Plate, Project.

### CODE REVIEW

**Comment by will-moore** [comment]
Travis failed with 
```
oving network scripts_default
The command ".omero/docker scripts" exited with 1.
```
Re-running...

File: omero/export_scripts/Batch_ROI_Export.py [hunk 1]
```diff
@@ -247,15 +247,36 @@ def link_annotation(objects, file_ann):
             o.linkAnnotation(file_ann)
 
 
+def get_images_from_plate(plate):
```
Comment (jburel) [inline omero/export_scripts/Batch_ROI_Export.py:250]: Note for self
This is the kind of method to add to "omero-py-toolbox"

File: omero/export_scripts/Batch_ROI_Export.py [hunk 2]
```diff
@@ -278,18 +299,19 @@ def batch_roi_export(conn, script_params):
...
+        objects = conn.getObjects(dtype, script_params['IDs'])
+        link_annotation(objects, file_ann)
     message = "Exported %s shapes" % len(export_data)
     return file_ann, message
 
 
 def run_script():
     """The main entry point of the script, as called by the client."""
-    data_types = [rstring('Dataset'), rstring('Image')]
+    data_types = [rstring(s) for s in
```
Comment (jburel) [inline omero/export_scripts/Batch_ROI_Export.py:313]: This is much more than Screen as indicated in the PR title.
All  types of containers need to be reviewed

Comment (will-moore) [reply omero/export_scripts/Batch_ROI_Export.py:313]: Yes, that is in the description. Not space in the title for all testing instructions.

**Comment by pwalczysko** [comment]
- run it on Project, Dataset, Plate, Screen, aslo in multiples
  - All works as expected, but couple of RFEs see below.


RFEs:

   - For plates, have a well (row, column, e.g. A1) as a parameter in the csv
   - For plates, the activities do not suggest "navigate to the plate where I just attached the csv" icon (unlike for P/Ds)
   - For multi-selections (select several Datasets or several Plates or several Projects or Screens, then run the script) or single Projects and Screens with multiple Datasets and Plates respectively, the parameter missing is the originating P/D/S/P

**Comment by jburel** [comment]
RFE https://github.com/ome/omero-scripts/issues/172

**Pull request merged**